selenium/che-selenium-test/src/test/resources/suites/CheOneThreadTestsSuite.xml

```diff
@@ -40,6 +40,8 @@
             <class name="org.eclipse.che.selenium.dashboard.ImportProjectFromGitHubTest"/>
             <class name="org.eclipse.che.selenium.factory.DirectUrlFactoryWithKeepDirectoryTest"/>
             <class name="org.eclipse.che.selenium.dashboard.workspaces.WorkspacesListTest"/>
+            <class name="org.eclipse.che.selenium.theia.TheiaBuildPluginTest"/>
```

Comment:
Why have to support `CheOneThreadTestsSuite.xml` suite as short as possible, because these tests nonscalable. 
What is the reason to move them from the `CheSuite.xml`? If they interfere each other, we can rearrange them inside the CheSuite.xml.

Reply:
The reason is if some theia stack is already run than each other workspace will be failed with errors

Reply:
In that case I would recommend to move second test far away (at the end) from first in CheSuite.xml list.

Reply:
Great catch, thanks